src/servers/authorization/email.ts

```diff
@@ -206,4 +218,97 @@ authRouter.post(
   }),
 )
 
+authRouter.post(
+  "/email/login/cookie",
+  wrapAsyncToRunInSpan({
+    namespace: "servers.middlewares.authRouter",
+    fnName: "emailLoginCookie",
+    fn: async (req: express.Request, res: express.Response) => {
+      baseLogger.info("/email/login/cookie")
+
+      const ipString = isProd ? req?.headers["x-real-ip"] : req?.ip
+      const ip = parseIps(ipString)
+
+      if (!ip) {
```

Comment:
let's add `recordExceptionInCurrentSpan()` for all the error case. that can be useful to debug. (like I start to do in this PR https://github.com/GaloyMoney/galoy/pull/2942/files)